src/main/java/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -1614,9 +1614,10 @@ public static XPathReference getAbsRef(XPathReference ref, TreeReference parentR
             tref = TreeReference.selfRef(); //only happens for <group>s with no binding
         }
 
+        TreeReference refPreContextualization = tref;

```

Comment:
@ctsims Assuming I'm missing something, but doesn't this just make `refPreContextualization` have a pointer to `tref`? How will this value remain different?